spotify_zlink.js

```diff
@@ -4,7 +4,7 @@ const path = require("path");
 const readline = require("readline");
 const cfg = require("./config.json");
 const query = '=`hm://album/v1/album-app/album/spotify:album:${e}/desktop?${[`catalogue=${encodeURIComponent(t.catalogue)}`,`locale=${encodeURIComponent(t.locale)}`,`username=${encodeURIComponent(t.username)}`].join("&")}`;' // What to look for in each line to inject code
-const splice = `if(!wSocket){console.log("Connecting to server...");wSocket=new WebSocket("${cfg.useSecure ? "wss" : "ws"}://${cfg.websocket.ipAddress}:${cfg.port}");wSocket.onopen=(event)=>{wSocket.onmessage=(msg)=>{msg=JSON.parse(msg.data);console.log(msg);switch(msg.req){case "playcount":if(msg.id.length!==22)return;const uri=\`hm://album/v1/album-app/album/spotify:album:\${msg.id}/desktop?\${[\`catalogue=\${encodeURIComponent(t.catalogue)}\`, \`locale=\${encodeURIComponent(t.locale)}\`].join("&")}\`;console.log(uri);a.default.resolver.get(uri,(e,t)=>{console.log(e,t);if(e){return wSocket.send(JSON.stringify({uuid:msg.uuid,success:!1,data:e}))}else{return wSocket.send(JSON.stringify({uuid:msg.uuid,success:!0,data:t.getJSONBody()}))}});break;case "hermes":const _uri=msg.uri;console.log(_uri);a.default.resolver.get(_uri,(e,t)=>{console.log(e,t);if(e){return wSocket.send(JSON.stringify({uuid:msg.uuid,success:!1,data:e}))}else{return wSocket.send(JSON.stringify({uuid:msg.uuid,success:!0,data:t.getJSONBody()}))}});break}}}}` // Inject with this string
+const splice = `if(!wSocket){console.log("Connecting to server...");wSocket=new WebSocket("${cfg.useSecure ? "wss" : "ws"}://${cfg.websocket.ipAddress}:${cfg.port}");wSocket.onopen=(event)=>{wSocket.onmessage=(msg)=>{msg=JSON.parse(msg.data);console.log(msg);switch(msg.req){case "playcount":if(msg.id.length!==22)return;const uri=\`hm://album/v1/album-app/album/spotify:album:\${msg.id}/desktop?\${[\`catalogue=\${encodeURIComponent(t.catalogue)}\`, \`locale=\${encodeURIComponent(t.locale)}\`].join("&")}\`;console.log(uri);i.default.resolver.get(uri,(e,t)=>{console.log(e,t);if(e){return wSocket.send(JSON.stringify({uuid:msg.uuid,success:!1,data:e}))}else{return wSocket.send(JSON.stringify({uuid:msg.uuid,success:!0,data:t.getJSONBody()}))}});break;case "hermes":const _uri=msg.uri;console.log(_uri);a.default.resolver.get(_uri,(e,t)=>{console.log(e,t);if(e){return wSocket.send(JSON.stringify({uuid:msg.uuid,success:!1,data:e}))}else{return wSocket.send(JSON.stringify({uuid:msg.uuid,success:!0,data:t.getJSONBody()}))}});break}}}}` // Inject with this string
```

Comment:
Difficult to tell due to the one line...but the change is this `a.default.resolver.get(uri` -> `i.default.resolver.get(uri`.